inductiva/molecules/scenarios/protein_solvation/README.md

```diff
@@ -0,0 +1,45 @@
+# Protein Solvation Scenario
+
+This scenario models the dynamics of a protein solvated in water inside a cubic box. The system is modelled with the motion equations for each individual atoms according to the prescription of a molecular force field approximation.
+
+To initilaize the scenario, the user can define the following parameters:
+- Protein .pdb file: PDB file containing the structure of the protein to simulate;
+- Temperature of the system in K.
+
+Now, the user is prepared to simulate protein dynamics. Please note that the scenario currently supports only PDB files as input. We have conducted extensive testing with [structures generated by the AlphaFold2 model](https://alphafold.ebi.ac.uk) and [structures from the RCSB database](https://www.rcsb.org). Before simulation, files undergo pre-processing to strip away all non-protein components of the system, such as ligands, non-standard residues, or water molecules, leaving only the protein.
+
+Also, right now the only supported simulator is `GROMACS`. Further developments will add new simulators to this scenario.
+
+The user can further define the following simulation parameters:
+- simulation_time_ns: The simulation time, in nanoseconds. Default value is 10.
+- output_timestep_ps: The time interval between recorded snapshots of the atom's positions in the trajectory output file, in picoseconds. Default value is 1.
+- integrator: The integrator to use for the simulation.
+    Options:
+                - "md" (Molecular Dynamics): Accurate leap-frog algorithm for
+                integrating Newton's equations of motion.
+                - "sd" (Steepest Descent): Stochastic dynamics integrator with
+                leap-frog scheme.
+                - "bd" (Brownian Dynamics): Euler integrator for Brownian or
+                position Langevin dynamics.
+    Default value if "md".
+- n_steps_min: Number of steps for energy minimization. Default value is 5000.
+
+Moreover, the hardware and interaction are configured with the usual general parameters - `machine_group`, `run_async`, `n_cores`.
+ Launching a simulation returns a task object, which can be used to verify the status of the simulation, get the simulation outputs and access post-processing tools. See more in [Tasks](inductiva/README.md).
+
+### Example
+
+```python
+ import inductiva
+
+ # Initialize the scenario
+ scenario = inductiva.fluids.scenarios.ProteinSolvation(
+     protein_pdb = "alanine.pdb"
+     temperature = 300)
+
+ # Run a simulation
+ task = scenario.simulate(simulation_time_ns = 20)
```

Comment:
I think you could add here the other options, for the purpose of explainability.